web-admin/src/features/dashboards/listing/utils.ts

```diff
@@ -0,0 +1,35 @@
+export function timeAgo(date: Date): string {
+  const now: number = new Date().getTime();
+  const diff: number = now - date.getTime();
+  const minute: number = 60 * 1000;
+  const hour: number = 60 * minute;
+  const day: number = 24 * hour;
+  const week: number = 7 * day;
```

Comment:
Nit: Why don't we use our existing methods and framework around time for this? (Luxon and Durations)

Reply:
Good idea to use `luxon` – just switched. But is this code redundant of something already in our codebase? I looked a little bit but didn't see a method that could be dropped in here.